ldms/src/ldmsd/ldmsd_stream.c

```diff
@@ -544,6 +544,7 @@ char * ldmsd_stream_client_dump()
 					first_client?"":",",
 					sym?sym[0]:_pbuf,
 					c->c_ctxt);
+			free(sym);
```

Comment:
This looks like a good fix, but maybe snuck into the wrong commit?

Reply:
ok, i pulled it out